business/templates/jobs.html

```diff
@@ -26,11 +27,30 @@ <h1 class="section-title">Job board</h1>
                 <a class="link-unstyled mt-3 d-block" href="{% url 'business_jobs_faq' %}"><p class="mb-0"><strong>Any questions? Check out our FAQ <i class="fas fa-arrow-right ml-1"></i></strong></p></a>
             </div>
         </div>
-        {% if not offers %}
-            <hr>
-            <p class="mt-4">There's no offers at the moment but make sure to check this page soon again!</p>
+
+        <div class='col-xs-3'>
+            <a href="{% url 'business_jobs' %}" class="link-unstyle">
+                <button type="button" class="btn btn-secondary rounded mt-1 {% if type == '' %}active{% endif %}"> Show all </button>
+            </a>
+
+            {% for k,v in types.items %}
+                <a class="link-unstyled" href="?type={{ k.value }}">
+                    <button type="button" class="btn btn-secondary rounded mt-1 {% if type == k.value|stringformat:"i" %}active{% endif %}"> {{ v }}</button>
+                </a>
+            {% endfor %}
+        </div>
+
+        {% if type == ''  and not offers %}
+                <hr>
+                <p class="mt-4">There's no offers at the moment but make sure to check this page soon again!</p>
+        {% elif not offers %}
+                <hr>
+                <p class="mt-4">For the selected category there are no offers at the moment, try and check out the others using the filters above. </p>
         {% endif %}
+
+
         {% for offer in offers %}
+            <br>
```

Comment:
Avoid `<br>` if possible, style it instead with something like `mt-3` for example.